From 23f17887e1fc96bc407afb13fda75f8442e9eae3 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 7 Jan 2018 17:24:42 +0200
Subject: [PATCH 2194/2241] fix: dts: set 8040-mcbin phy-mode of mac3 in cp1 to
 1000base-x

CP110-1 mac3 phy-mode fixed to correct "1000base-x"

Change-Id: I6b3c0488b2591cc4f7dbf563809302ef783b7d06
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48611
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 5bc75fd..484e7dc 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -247,7 +247,7 @@
 					phy = <&smi_phy0>;
 				};
 				cp1_emac3: mac3 {
-					phy-mode = "sgmii"; /* lane-5 */
+					phy-mode = "1000base-x"; /* lane-5 */
 					phy-speed = <2500>;
 					phys = <&cp1_comphy 5 COMPHY_HS_SGMII2>;
 					phy-names = "comphy";
-- 
2.7.4

